students/etwum/lesson04/mailroom_part2.py

```diff
@@ -0,0 +1,153 @@
+#!/usr/bin/env python3
+
+
+# initial list of donors
+lst_Donors = [{"donor name": "Lionel Messi", "total donations": 1000000.00, "number donations": 5, "avg donation": 20000.00},
+              {"donor name": "Thierry Henry", "total donations": 500, "number donations":1, "avg donation": 500},
+              {"donor name": "Michael Jordan", "total donations": 45000, "number donations": 3, "avg donation": 15000},
+              {"donor name": "Kobe Bryant", "total donations": 8000, "number donations": 2, "avg donation": 4000}]
```

Comment:
you should really re-consider converting this data structure to a dict rather than a list

Reply:
what are the pros and cons of using nested dictionaries? In the foundation course we learned to use the nested dictionary to create tables of info

https://www.youtube.com/watch?v=Nsyw1W4zQhw&index=5&list=PLfycUyp06LG9I4194n8OvgCbmEXQt3a8z

Reply:
can you give me an example of what you really mean by nested dictionary?

(I don't want to make assumptions here)

Reply:
I mean nested in the sense that they are dictionaries within a list